MatrixSDK/Room/Polls/PollAggregator.swift

```diff
@@ -149,14 +162,33 @@ public class PollAggregator {
         reloadPollData()
     }
     
-    private func reloadPollData() {
+    public func reloadPollData() {
         delegate?.pollAggregatorDidStartLoading(self)
         
         session.aggregations.referenceEvents(forEvent: pollStartEventId, inRoom: room.roomId, from: nil, limit: -1) { [weak self] response in
-            guard let self = self else {
+            guard let self else {
                 return
             }
             
+            guard let event = response.originalEvent else {
+                // we didn't found the start event
+                self.delegate?.pollAggregator(self, didFailWithError: PollAggregatorError.invalidPollStartEvent)
+                return
+            }
+            
+            // if we don't already have a pollStartedEvent, update it
+            if pollStartedEvent == nil {
```

Comment:
What happens if we remove this if?

Reply:
Everything works fine. I removed it (it was no longer necessary, as the poll is now optional)